### PR TITLE
Add simple default log formatter

### PR DESCRIPTION
**Pull request by Doikor**
Need some suggestions about the actual formats. How to show the enclosing? (what kind of separator?)

### CODE REVIEW

File: core/src/main/scala/io/odin/formatter/Formatter.scala [hunk 1]
```diff
@@ -10,6 +12,7 @@ trait Formatter {
 
 object Formatter {
 
-  val simple: Formatter = (msg: LoggerMessage) => s"${msg.toString}. Message: ${msg.message()}"
+  val simple: Formatter = (msg: LoggerMessage) => p"${msg.toString}. Message: ${msg.message()}"
+  val default: Formatter = (msg: LoggerMessage) => p"${msg.timestamp.t.F} [${msg.threadName}] ${msg.level.show} ${msg.position.packageName}.${msg.position.fileName}:${msg.position.line} - ${msg.message()}"
```
Comment (sergeykolbasov) [inline core/src/main/scala/io/odin/formatter/Formatter.scala]: It's missing stack trace information in case if throwable is there

**Comment by sergeykolbasov** [comment]
How about to print `enclosure:lineNumber` instead of `fileName:lineNumber`? I think it's more useful in logs when you see the actual method that logged something. Although, in `for .. yield` comprehension it usually prints out `$anon` as the last part of the enclosure, I guess due to the de-sugaring of comprehension into a chain of `flatMap` and anonymous lambdas.

**Comment by Doikor** [comment]
Yeah was not so sure about the format. I still want the file name there so need to come up with another separator. Maybe `package.filename$enclosure:line` ?

**Comment by sergeykolbasov** [comment]
Hm, `enclosure` is actually the full name, so it'll contain a package name as well.
Also, file name contains the absolute path to the scala file, so this information is overkill for logs in my opinion.

File: core/src/main/scala/io/odin/formatter/Formatter.scala (hunk 1 above)
Comment (Doikor) [reply core/src/main/scala/io/odin/formatter/Formatter.scala]: Would we want that on the same row or throw a newline in there? Personally I would not print the exception here at all but instead make it the responsibility of whoever is logging to put it in the message (And remove the throwable from the log functions)

**Pull request merged**